ci-operator/step-registry/gather/libvirt/gather-libvirt-commands.sh

```diff
@@ -0,0 +1,390 @@
+#!/bin/bash
+
+set -o nounset
```

Comment:
1) Is this step used somewhere? It does not seem to be inlucded in the `post` secrion of the workflow
2) `gather` step does not sound like something that needs to have a duplicate, `libvirt` specific flavor. Could we use the shared `gather` step?

Reply:
> 1. Is this step used somewhere? It does not seem to be inlucded in the `post` secrion of the workflow
> 2. `gather` step does not sound like something that needs to have a duplicate, `libvirt` specific flavor. Could we use the shared `gather` step?

The libvirt requires dnsConfig in ref to connect to the cluster. Without the connection gather step will be useless.